.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,82 @@
+name: Bug report
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Fill in this template to report a bug
+
+title: "[BUG] "
+
+labels: ["bug"]
+
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description:
+        Please search to see if an issue already exists for the bug you
+        encountered.
+      options:
+        - label: I have searched the existing issues
+
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: Describe what outcome you expected.
+      placeholder: Describe what outcome you expected.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: Describe what you got instead.
+      placeholder: Describe what you got instead.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Error message
+      description: If possible paste below the error message you encountered.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        If necessary give information about your setup.
+      value: |
+        - OS:
+        - Python version:
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Version
+      description: |
+        Which version did you encounter this bug on?
```

Comment:
:| 
So far none of the tools have been released / versioned yet. That's another pressing issue to get to. I feel like we keep this section and just figure out the releases next.

Reply:
OK I will comment this out until we have at least one versioned package

Reply:
done